src/components/views/Agreement.js

```diff
@@ -165,12 +195,13 @@ export default class Agreement extends React.Component {
       helperApp
     } = this.props;
 
-    if (isLoading) return <LoadingPane defaultWidth="60%" onClose={handlers.onClose} />;
+    const { showDuplicateAgreementModal } = this.state;
 
+    if (isLoading) return <LoadingPane defaultWidth="60%" onClose={handlers.onClose} />;
     return (
       <React.Fragment>
         <Pane
-          actionMenu={this.getActionMenu}
+          actionMenu={({ onToggle }) => this.getActionMenu(onToggle)}
```

Comment:
Let's just pass the entire object like normal (ie, leave this line as-is on `master`), no need to destructure it here when it costs us an extra function definition each render _and_ future flexibility.